src/routes/posts.js

```diff
@@ -18,15 +18,25 @@ const fetchPosts = async (req, res, next) => {
 
 		res.status(200).json({
 			status: 'success',
-			data: feed.map((article) => ({
-				id: article.id,
-				userId: article.userId,
-				title: article.title,
-				content: article.content,
-				image: article.image,
-				published: article.published,
-				createdOn: article.createdAt
-			}))
+			data: feed.map((post) => {
+				if(post.type === 'article')
+					return {
+						id: post.id,
```

Comment:
the consistency reason is fine. but if we want it consistent, we can go with the existing transform functions we have in the `articles.js` and `gifs.js`

1. Kindly create a folder called `common` in the `routes/` folder
2. Have a file called `tranformers.js` or something better
3. move the `transformArticleResponse` and `transformGifResponse` from the `articles.js` and `gifs.js` respectively
4. Export the functions from there and use it in the `articles.js`, `gifs.js` and `posts.js`

that way we get the consistency and also get a single place for them when something is getting changed

Reply:
Yes please, I will make the change

Reply:
done

Reply:
nice